parts/k8s/cloud-init/masternodecustomdata.yml

```diff
@@ -118,6 +118,26 @@ write_files:
   content: !!binary |
     {{CloudInitData "aptPreferences"}}
 
+- path: /etc/apt/apt.conf.d/10periodic
```

Comment:
Do we need this `10periodic` configuration file? I would think that simply removing the `99periodic` configuration with all the `0` values would restore the default unattended upgrade configuration. Are you also suggesting we explicitly re-configure the default values for `Update-Package-Lists`, `Download-Upgradeable-Packages`, `AutocleanInterval`, and `Unattended-Upgrade`?

Reply:
I'll come back to you about that cause it's been a while and I don't remember where i've taken the content of `10periodic`

Reply:
This `10periodic` seems to be part of the `update-notifier-common` package.

I guess we don't need to ship it ourselves.